docs/sources/datasources/mssql.md

```diff
@@ -23,7 +23,7 @@ To access data source settings, hover your mouse over the **Configuration** (gea
 | `Authentication` | Authentication mode. Either using SQL Server Authentication or Windows Authentication (single sign on for Windows users).             |
 | `User`           | Database user's login/username                                                                                                        |
 | `Password`       | Database user's password                                                                                                              |
-| `Encrypt`        | This option determines whether or to which extent a secure SSL TCP/IP connection will be negotiated with the server, default `false`. |
+| `Encrypt`        | This option determines whether or to which extent a secure SSL TCP/IP connection will be negotiated with the server, default `true`. |
```

Comment:
```suggestion
| `Encrypt`        | This option determines if and to what extent a secure SSL TCP/IP connection will get negotiated with the server. The default value is`true`.  |
```

Reply:
I've added that suggestion @achatterjee-grafana, thanks.